src/adam_bashforth.jl

```diff
@@ -1,57 +1,76 @@
-module Tmp
 ####################################
-# Explicit Adam-Bashforth solvers
+# Implicit Adam-Bashforth solvers
 ####################################
 # (Hairer & Wanner 1996, Vol I, p.357-358
-include("ODE.jl")
 
-##Fixed step, fixed order, multistep explicit method
 
-function ode_ab(F,y0, tspan,order ::Integer)
-    if !(2 <= order <= 4)
-        error("Currently only orders 2,3, and 4 are implemented for Adam-Bashforth method")
+##Function: ode_imp_ab(F,y0, tspan, order(optional))
+##order is set to 4 by default
+##Adam Bashforth is a fixed step, fixed order, multistep Implicit method
+function ode_imp_ab(F::Function,y0, tspan,order=4 ::Integer)
```

Comment:
`order=4 ::Integer` does not what you think it should: it checks that 4 is an Integer, then assigns that to `order`. You want `order::Integer=4`.